rs/execution_environment/src/canister_manager/tests.rs

```diff
@@ -3912,68 +3447,6 @@ fn create_canister_when_compute_capacity_is_oversubscribed() {
         );
 }
 
-#[test]
-fn install_code_when_compute_capacity_is_oversubscribed() {
-    let mut test = ExecutionTestBuilder::new()
-        .with_allocatable_compute_capacity_in_percent(0)
-        .build();
-    let canister_id = test.create_canister(Cycles::new(2_000_000_000_000_000));
-
-    // Manually set the compute allocation higher to emulate the state after
-    // replica upgrade that decreased compute capacity.
-    test.canister_state_mut(canister_id)
-        .scheduler_state
-        .compute_allocation = ComputeAllocation::try_from(60).unwrap();
-
-    // Updating the compute allocation to a higher value fails.
-    let err = test
-        .install_canister_with_allocation(
-            canister_id,
-            UNIVERSAL_CANISTER_WASM.to_vec(),
-            Some(61),
-            None,
-        )
-        .unwrap_err();
-    err.assert_contains(
-        ErrorCode::SubnetOversubscribed,
-        "Canister requested a compute allocation of 61% \
-        which cannot be satisfied because the Subnet's \
-        remaining compute capacity is 60%.",
-    );
-
-    // Updating the compute allocation to the same value succeeds.
-    test.install_canister_with_allocation(
-        canister_id,
-        UNIVERSAL_CANISTER_WASM.to_vec(),
-        Some(60),
-        None,
-    )
-    .unwrap();
-    assert_eq!(
-        ComputeAllocation::try_from(60).unwrap(),
-        test.canister_state(canister_id)
-            .scheduler_state
-            .compute_allocation
-    );
-
-    test.uninstall_code(canister_id).unwrap();
```

Comment:
This uninstall should not be necessary for the sake of this test.

Reply:
I think we drop this test anyway so it shouldn't matter.